src/main/java/duke/core/Parser.java

```diff
@@ -43,12 +43,11 @@ public Parser(String userInput) {
      * @return .
      * @throws DukeException .
      */
-    public String[] parseDeletePatientTask() throws DukeException {
-        String[] formattedInput = new String[2];
+    public String parseDeletePatientTask() throws DukeException {
```

Comment:
Probably should not be doing any alterations in the Parser--I've implemented the class so that it handles multiple words, like `delete patient task`, and the miscommunication between us is causing unnecessary bugs. Just give me a heads up of your desired input for the Command, and I'll implement it for you.

Reply:
noted